to_sandbox.txt

```diff
@@ -35,5 +35,5 @@ ofl/ibmplexmono # https://github.com/google/fonts/pull/5843
 ofl/ibmplexsans # https://github.com/google/fonts/pull/5843
 ofl/ibmplexserif # https://github.com/google/fonts/pull/5843
 
-# Lang
-lang/languages/th_Thai.textproto
+# Sample texts
+lang/languages/th_Thai.textproto # https://github.com/google/fonts/pull/5925
```

Comment:
@m4rc1e This path is probably the one breaking the CI, although I was sure you made fixed for that last year